File: 백준/Bronze/30802. 웰컴 키트/웰컴 키트.py
import math

sum_value = 0

n = int(input())
size_amount = map(int, input().split())
t,p = map(int, input().split())

for value in size_amount:
    sum_value += math.ceil(value / t)

print(sum_value)
print(f"{n // p} {n % p}")

File: 백준/Bronze/30802. 웰컴 키트/웰컴 키트.go
package main

import (
	"fmt"
	"math"
)

func main() {
	var n, t, p int
	var sizeAmount []int

	fmt.Scan(&n)

	var size int
	for i := 0; i < 6; i++ {
		if _, err := fmt.Scan(&size); err != nil {
			break
		}
		sizeAmount = append(sizeAmount, size)
	}

	fmt.Scan(&t, &p)

	sumValue := 0
	for _, value := range sizeAmount {
		sumValue += int(math.Ceil(float64(value) / float64(t)))
	}
	
	fmt.Println(sumValue)
	fmt.Printf("%d %d\n", n/p, n%p)
}